app/client/components/transferAccount/transferAccountManager.jsx

```diff
@@ -317,55 +314,6 @@ class TransferAccountManager extends React.Component {
                   <ManagerText>{this.state.one_time_code}</ManagerText>
                 </SubRow>
               </Row>
-              <Row style={{ margin: "0em 1em" }}>
```

Comment:
we haven't used this in two years. makes sense to remove from frontend and reimplement using ant-design if needed in future.

Reply:
🐜